runner_manager/jobs/reset.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+from redis_om import NotFoundError
+
+from runner_manager import RunnerGroup
+from runner_manager.clients.github import GitHub
+from runner_manager.dependencies import get_github
+
+log = logging.getLogger(__name__)
+
+
+def group(pk: str):
+    """Job to reset a runner group.
+
+    Args:
+        pk (str): Runner group primary key
+    """
+    github: GitHub = get_github()
+    try:
+        group = RunnerGroup.get(pk)
+        group.reset(github)
```

Comment:
Same comment as in https://github.com/scality/runner-manager/pull/365 regarding the try statement containing lines that are not catchable by the except, so they should be put outside of it